pkgs/top-level/all-packages.nix

```
@@ -5955,6 +5955,8 @@ in
 
   libtorrent = callPackage ../tools/networking/p2p/libtorrent { };
 
+  jesec-libtorrent = callPackage ../tools/networking/p2p/jesec-libtorrent { };
```

Wouldn't
```suggestion
  libtorrent-jesec = callPackage ../tools/networking/p2p/libtorrent-jesec { };
```
make more sense given there is `libtorrent-rasterbar`? What was your reasoning behind the name?

---

My reasoning was that I used `jesec-rtorrent` as `(github.com/)jesec/rtorrent`, so I just copied the same naming scheme to `jesec/libtorrent`. I think you're right though, I should change the name of `jesec-libtorrent`. I'll probably do the same for `jesec-rtorrent` as well, do you think that would be better for consistency purposes? (I think it will be so it matches `libtorrent`, plus there's at least one more rTorrent fork that uses the `rtorrent-*` naming scheme.)

---

> I'll probably do the same for `jesec-rtorrent` as well, do you think that would be better for consistency purposes?

Yes, perfect.